include/RAJA/pattern/forall.hpp

```diff
@@ -127,7 +188,7 @@ template <typename EXEC_POLICY_T, typename LOOP_BODY>
 RAJA_INLINE void forall_Icount(const IndexSet& c, LOOP_BODY loop_body)
```

Comment:
Do we have a clear policy when we should use reference vs. universal reference for method args? We probably should and document it in the coding guidelines that I will eventually get back to...